File: classes/parte-5-funciones-y-modularidad/086-modulos-paquetes-y-espacios-de-nombres/implementaciones/c/main.c
#include <stdio.h>

/* En C la modularidad se hace por archivos .h/.c; aquí una función local. */
long doble(long n) {
    return 2 * n;
}

int main(void) {
    long n;
    if (scanf("%ld", &n) != 1) return 1;
    printf("resultado=%ld\n", doble(n));
    return 0;
}

File: classes/parte-5-funciones-y-modularidad/086-modulos-paquetes-y-espacios-de-nombres/implementaciones/php/main.php
<?php
// PHP usa namespaces; aquí una función que actúa como utilidad del módulo.
function matematicas_doble($n) {
    return 2 * $n;
}

$n = (int) trim(fgets(STDIN));
echo "resultado=" . matematicas_doble($n) . "\n";
